types/auth0-js/index.d.ts

```diff
@@ -347,6 +347,8 @@ export class Redirect {
             password: string,
             /** name of the connection where the user will be created */
             connection: string,
+            /** allow user_metadata for signup */
+            userMetadata?: DbSignUpOptions
```

Comment:
The docs you linked, and your own comment, seem to indicate the field name is `user_metadata`, not `userMetadata`.

Reply:
Hey, thanks for your review. You're right regarding `signUp` property but `signUpAndLogin` uses cameCase instead (as in https://auth0.github.io/auth0.js/global.html#loginWithCredentials).

Reply:
Actually, there's something weird because the `userMetadata` is already in `signupAndAuthorize` test but is missing in the typings 🤔